src/frontend/js/components/AddressesManagement/AddressForm.tsx

```diff
@@ -41,6 +42,22 @@ const AddressForm = ({ handleReset, onSubmit, address }: Props) => {
   const [languageCode] = intl.locale.split('-');
   const countryList = countries.getNames(languageCode);
 
+  const getLocalizedErrorMessage = (
+    error: Maybe<
+      | string
+      | {
+          key: ErrorKeys;
+          values: Record<PropertyKey, string | number | Array<string | number>>;
+        }
+    >,
+  ) => {
+    if (!error) return undefined;
+    if (typeof error === 'string') return error;
```

Comment:
```suggestion
    if (typeof error === 'string') return error;
```
I don't think this case is tested, it could be great to add a test for this purpose

Reply:
Finally, I returned a default localized message in the case the error message provided does not have corresponding i18n message.

Reply:
we're just missing the test when a translation is missing

Reply:
Good catch ! It gave me a hard time but it's now tested : )

Reply:
really good one!